src/boundary/BoundaryData.h

```diff
@@ -33,32 +33,33 @@ enum BoundaryCondition : int {
 };
 
 class BoundaryData {
-public:
+ public:
     BoundaryData();
     ~BoundaryData();
     void print();
 
-    static std::string getFieldTypeName(FieldType f);
-    static std::string getBoundaryConditionName(BoundaryCondition bc);
-    static std::string getPatchName(Patch p);
+    static std::string get_field_type_name(FieldType f);
+    static std::string get_boundary_condition_name(BoundaryCondition bc);
+    static std::string get_patch_name(Patch p);
```

Comment:
can be const

Reply:
static member cannot have a const qualifier